packages/suite/src/reducers/wallet/sendFormReducer.ts

```diff
@@ -79,7 +79,8 @@ export default (state: State | null = null, action: WalletAction): State | null
                     return draft;
                 }
 
-                if (!isAddressValid(action.address, symbol)) {
+                // turn off validation for bitcoin like coins - handled by connect compose transaction
+                if (!isAddressValid(action.address, symbol) && networkType !== 'bitcoin') {
```

Comment:
this should be flipped
`if (networkType !== 'bitcoin' && !isAddressValid(action.address, symbol))`
this way isAddressValid fuction will not be called on bitcoin addresses